index.html

```diff
@@ -1,116 +1,182 @@
+                        <span class="button gallery-button gallery-button-next">Вперёд</span>
+        </section>
+
+        <section class="contacts">
+                    
+                    <h2>
+                       Контактная информация
+                    </h2>
+
+                    <p>
+                        
+                       Тут инфа<br>
+                       ещё инфа<br>
+                       инфа<br>
+                       инфа<br>
+                       инфа
+                   
+                    </p>
+
     <a class="button" href="map.html">Как проехать</a>
     <a class="button" href="contacts.html">Обратная связь</a>
+        
 </section>
-<section class="appointment">
-    <h2>Записаться</h2>
-    <p class="appointment-info">Укажите желаемую дату и время и мы свяжемся с вами для подтверждения говна</p>
+        
+        <section class="appointment">
+                    
+            <h2>
+                Записаться
+            </h2>
+
+    <p class="appointment-info">Укажите желаемую дату и время и мы свяжемся с вами для подтверждения</p>
+    
     <!--Тут будет канеки кек(форма)-->
-</section>
-  </main>
-  <footer class="main-footer">
-      <p class="footer-contacts">
-          контакты, ну брки можно сюда пристроить ещё, шоб всё по красоте и в панельку вместилось
-      </p>
-      <div class="footer-social">
-          <b>Давайте дружить щещенАмы</b>
-          <ul>
-          <a class="social-button" href="линк">VK</a>
-          <a class="social-button" href="линк">FB</a>
-          <a class="social-button" href="линк(не игорь)">INST</a>
-        </ul>
-      </div>
-      <p class="footer-copyright">
-      <b>Разработанно:</b>
-      <a class="button" href="https://osu.ppy.sh/users/kotakbas">Апнул 7к пп</a>
-      </p>
-  </footer>
-
-  <section class="media modal-login">
-      <h2>Личный кабинет</h2>
-      <p>Введите логин и пароль.</p>
+
+        </section>
+    </main>
+  
+    <footer class="footer">
+                    <p class="footer-contacts">контакты, ну брки можно сюда пристроить ещё, шоб всё по красоте и в панельку вместилось</p>
+                        
+                <div class="footer-social">
+                    
+                    <b>
+                        Давайте дружить!
+                    </b>
+            <ul>
+                    <a class="social-button" href="линк">VK</a>
+                    <a class="social-button" href="линк">FB</a>
+                    <a class="social-button" href="линк(не игорь)">INST</a>
+            </ul>
+                </div>
+                    <p class="copyright">
+                    
+                        <b>
+                            Разработанно:
+                        </b>
+
+                    <a class="button" href="https://osu.ppy.sh/users/kotakbas">Апнул 7к пп</a>
+                    
+                    </p>
+    </footer>
```

Comment:
те же проблемы
Плюс, если мы делаем список, в нем должны быть элементы списка
```suggestion
    <footer class="footer">
        <p class="footer-contacts">контакты, ну брки можно сюда пристроить ещё, шоб всё по красоте и в панельку вместилось</p>
                        
        <div class="footer-social">
             <b>
                 Давайте дружить!
             </b>
             
             <ul class="social-list">
                    <li class="social-item">
                        <a class="social-link" href="линк">VK</a>
                    </li>
                    <li class="social-item">
                        <a class="social-link" href="линк">FB</a>
                    </li>
                    <li class="social-item">
                        <a class="social-link" href="линк(не игорь)">INST</a>
                    </li>
             </ul>
         </div>
         <p class="copyright">
             <b>Разработанно: </b> 
             <a class="button" href="https://osu.ppy.sh/users/kotakbas">
                 Апнул 7к пп
             </a>
         </p>
    </footer>
```